alacritty.yml

```diff
@@ -218,11 +218,11 @@
     # Allowed values are CellForeground/CellBackground, which reference the
     # affected cell, or hexadecimal colors like #ff00ff.
     #matches:
-    #  foreground: '#000000'
+    #  foreground: '#ff0000'
```

Comment:
I've told you that I don't want to change this, didn't I? I'm not sure how to make myself more clear on how I'd suggest to resolve this.

Reply:
There is zero reason to change the defaults in this PR. The requested change is completely unrelated to that. I'm not sure why you're derailing this PR with an unrelated change.